### ISSUE TITLE
can not put barrels in the car 

### ISSUE THREAD

**Issue by toddy617** [LuaDeldu/esx_yellowjackjob#2]
Hey this looks like a really nice job but the blip on the cars do not work to put barrels in have you got a fix?

**Comment by japael** [comment]
yeah i'm noticing the same error, trying to fix but cant see how.

**Comment by toddy617** [comment]
I have looked and looked and can't get my head around it shame as everything else works and seems such a nice bit of work

**Comment by Snoopydrop34** [comment]
bonjour le probleme et toujour dispo car je ne peux pas mettre dans la voiture serait t'il possible de le corriger merci a vous

**Comment by BBisWicked** [comment]
Same here! We are really checking the code day and night, we just can't fix it, any clues?
